Src/FluentAssertions/CallerStatementBuilder.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using FluentAssertions.CallerIdentification;
+
+namespace FluentAssertions
+{
+    internal class CallerStatementBuilder
```

Comment:
🔧 I would move this file to the `CallerIdentification` folder as well. It's an implementation detail of the `CallerIdentifier`

Reply:
Done